exasol_advanced_analytics_framework/event_context/udf_event_context.py

```diff
@@ -9,61 +9,71 @@
     ColumnType
 
 from exasol_advanced_analytics_framework.event_context.event_context_base \
-    import EventContextBase
+    import EventContext, Row
 
 
-class UDFEventContext(EventContextBase):
-    def __init__(self, ctx, exa, column_mapping: Mapping[str, str],
-                 start_col: int = 0):
-        super().__init__(ctx)
-        self.start_col = start_col
-        if not isinstance(column_mapping, OrderedDict):
-            raise ValueError(
-                f"column_mapping needs to be a OrderedDict, "
-                f"got {type(column_mapping)}")
-        self.column_mapping = column_mapping
-        self.original_columns = list(self.column_mapping.keys())
-        self.new_columns = list(self.column_mapping.values())
-        self.__ctx = ctx
-        self.exa = exa
+class UDFEventContext(EventContext):
 
-    def _get_mapped_column(self, original_name: str) -> str:
-        if original_name in self.column_mapping:
-            return self.column_mapping[original_name]
-        raise ValueError(
-            f"Column {original_name} does not exists "
-            f"in mapping {self.column_mapping}")
+    def __init__(self, ctx, exa, column_mapping: OrderedDict[str, str],
+                 start_col: int = 0):
+        self._start_col = start_col
+        self._ctx = ctx
+        self._has_next = True
+        print("column_mapping", column_mapping)
```

Comment:
better to either remove debug print or replace it with logger

Reply:
thx, overlooked it